src/payment/strategies/ppsdk/ppsdk-payment-processor.ts

```diff
@@ -0,0 +1,18 @@
+import { OrderPaymentRequestBody } from '../../../order';
+import { PPSDKPaymentMethod } from '../../ppsdk-payment-method';
+
+import { initializeNone, isNone } from './initializers';
+
+export type PaymentProcessor = (payment: OrderPaymentRequestBody | undefined) => Promise<void>;
```

Comment:
It is a bit of a catch all.

Each payment processor will kick off the initial call to the `/payments` processing API, then handle its response(s) (via the `stepHandler`). Once all the responses have been dealt with it will terminate in either success (resolving) or error/failure (rejecting).

Given that success signifies that BigPay has updated the order to 'paid' (side-effect), we don't really have anything to return.